src/Document/EditableDialogBox/LayoutItem/TabPanelItem.php

```diff
@@ -1,14 +1,14 @@
 <?php
 declare(strict_types=1);
 
-namespace Neusta\Pimcore\EditorConfigBundle\Document\EditableDialogBox\LayoutItem;
+namespace Neusta\Pimcore\AreabrickConfigBundle\src\Document\EditableDialogBox\LayoutItem;
 
-use Neusta\Pimcore\EditorConfigBundle\Document\EditableDialogBox\LayoutItem;
+use Neusta\Pimcore\AreabrickConfigBundle\src\Document\EditableDialogBox\LayoutItem;
 
 class TabPanelItem extends LayoutItem
 {
     /**
-     * @param list<PanelItem> $items
+     * @param PanelItem $items
```

Comment:
That is wrong. It was right before.

Reply:
I don't know what the refactorer did with it but now it's corrected - thx for your strong view.